pkg/controller/installation/core_controller.go

```diff
@@ -456,6 +467,17 @@ func (r *ReconcileInstallation) Reconcile(request reconcile.Request) (reconcile.
 		return reconcile.Result{}, err
 	}
 
+	// A status is needed at this point for operator scorecard tests.
+	// status.variant is written later but for some tests the reconciliation
+	// does not get to that point.
+	if reflect.DeepEqual(status, operator.InstallationStatus{}) {
```

Comment:
Why wait till here to do this status update? Why not do it right after the `if` block you added above?

Reply:
I figure we should wait until the merged/filled installation is written first before updating the status.